doc/_sphinx/conf.py

```diff
@@ -17,7 +17,7 @@
 # -- Project information -----------------------------------------------------
 
 project = 'Flame'
-copyright = '2021, Blue Fire Team'
+copyright = '2021-2023, Blue Fire Team'
```

Comment:
Maybe the year could be skipped here entirely? We don't really care about the copyright anyways since it's MIT licenced.

Reply:
Removed.

Reply:
General suggestion ([see here](https://www.copyrightlaws.com/copyright-symbol-notice-year/#:~:text=Which%20Year%20Should%20You%20Use,to%20the%20public%20without%20restriction.)) is to only write the year of first publication, not the range.